scripts/verify-nupkgs.ps1

```diff
@@ -16,10 +16,10 @@ function Verify-Nuget-Packages($packageDirectory)
                      "Microsoft.NET.Test.Sdk" = 13;
                      "Microsoft.TestPlatform" = 421;
                      "Microsoft.TestPlatform.Build" = 19;
-                     "Microsoft.TestPlatform.CLI" = 303;
+                     "Microsoft.TestPlatform.CLI" = 305;
```

Comment:
>305 [](start = 52, length = 3)

Can you please mention what is difference here ?

Reply:
In Microsoft.TestPlatform.CLI
    1. System.Memory.dll is removed(not coming any more)
    2. System.Security.Principal.Windows.dll is coming in these folders.
         a. contentFiles/any/netcoreapp2.1/runtimes/unix/lib/netcoreapp2.0/System.Security.Principal.Windows.dll
         b. contentFiles/any/netcoreapp2.1/runtimes/win/lib/netcoreapp2.0/System.Security.Principal.Windows.dll
         c. contentFiles/any/netcoreapp2.1/System.Security.Principal.Windows.dll

In Microsoft.TestPlatform.ObjectModel
 netstandard1.4 which contained 3 files (CoreUtilities.dll, PlatformAbstractions.dll, ObjectModel.dll) has been removed

In Microsoft.TestPlatform.Portable, these have been removed
1. tools/netcoreapp2.1/System.Memory.dll
2. tools/netcoreapp2.1/TestHost/System.Runtime.InteropServices.RuntimeInformation.dll